tez-common/src/main/java/org/apache/tez/dag/records/DAGIDAware.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.tez.dag.records;
+
+import org.apache.hadoop.yarn.api.records.ApplicationId;
+
+public interface DAGIDAware {
+    TezDAGID getDAGId();
```

Comment:
even if it looks strange (too many capitals), this might be changed to
```
getDAGID();
```
to stay consistent with others, so it should suggest that a TezID Object is returned here which is "ID", not an integer, which is id

Reply:
done